src/main/java/jnr/constants/platform/fake/OpenFlags.java

```diff
@@ -27,7 +27,7 @@
 private final long value;
 private OpenFlags(long value) { this.value = value; }
 public static final long MIN_VALUE = 1L;
-public static final long MAX_VALUE = 0x80000L;
```

Comment:
Both old and new MAX is weird....Really confused what it is about now.

Reply:
I believe these two were added, which bumped up the bits:

```
O_TMPFILE(0x100000),
O_CLOEXEC(0x200000);
```